institutions-api/util/ror_utils.py

```diff
@@ -0,0 +1,11 @@
+from typing import Optional
+from fastapi import HTTPException
+import requests
+from ..models.api_models import ROR_ID_PREFIX
+
+
+def validate_ror_id(ror_id: Optional[str]):
+    """ Check whether an ROR ID is valid by performing a HEAD request to ror.org """
+    # TODO we might want to rate-limit this in some way
+    if ror_id and requests.head(ror_id).status_code != 200:
```

Comment:
```suggestion
    if ror_id and requests.head(ror_id, allow_redirects=True).status_code != 200:
```
According to https://requests.readthedocs.io/en/latest/user/quickstart/#redirection-and-history, HEAD does not follow redirects by default.